pkg/federated/parseResponse.go

```diff
@@ -38,8 +38,8 @@ type Data struct {
 }
 
 type GraphQLPayload struct {
-	Data   Data    `json:"data"`
-	Errors []error `json:"errors,omitempty"`
+	Data   Data     `json:"data"`
+	Errors []string `json:"errors,omitempty"`
```

Comment:
Updated to store error strings as error type interface was causing issues during unmarshaling. Please review.